tests/clients/test_base.py

```diff
@@ -16,24 +16,23 @@
 from collections import namedtuple
```

Comment:
Should we also add a test for the `ErrorSchema` as discussed?

Reply:
Do you think it's worth it? It's 2 optional strings

Reply:
👍